golang/cosmos/app/app.go

```diff
@@ -530,27 +528,14 @@ func (app *GaiaApp) MustInitController(ctx sdk.Context) {
 	}
 	app.controllerInited = true
 
-	var bootstrapAddr sdk.AccAddress
-	gs := app.VpurseKeeper.GetGenesis(ctx)
-	if len(gs.BootstrapAddress) > 0 {
-		ba, err := sdk.AccAddressFromBech32(gs.BootstrapAddress)
-		if err != nil {
-			fmt.Fprintln(os.Stderr, "Cannot get bootstrap addr", err)
-			os.Exit(1)
-		}
-		bootstrapAddr = ba
-	}
-
 	// Begin initializing the controller here.
 	action := &cosmosInitAction{
-		Type:             "AG_COSMOS_INIT",
-		VPursePort:       app.vpursePort,
-		IBCPort:          app.ibcPort,
-		StoragePort:      swingset.GetPort("storage"),
-		ChainID:          ctx.ChainID(),
-		BootstrapAddress: bootstrapAddr.String(),
-		BootstrapValue:   gs.BootstrapValue.String(),
-		DonationValue:    gs.DonationValue.String(),
+		Type:        "AG_COSMOS_INIT",
```

Comment:
Keep the field order the same as in the struct definition above, for easier inspection. Alphabetical unless there is a compelling reason for something else.